crates/client/assets/schema.sdl

```diff
@@ -970,7 +970,7 @@ type Query {
 		"""
 		owner: Address,		first: Int,		after: String,		last: Int,		before: String
 	): MessageConnection!
-	messageProof(transactionId: TransactionId!, nonce: Nonce!, commitBlockId: BlockId, commitBlockHeight: U32): MessageProof
+	messageProof(transactionId: TransactionId!, nonce: Nonce!, commitBlockId: BlockId, commitBlockHeight: U32): MessageProof!
```

Comment:
@acerone85 @luizstacio @LuizAsFight is it a breaking change for you?

If yes, we can just always return `Some(proof)`.

Reply:
This is not a breaking change for the TS SDK, we have null handling but it'll just be redundant.

Not sure about the FE apps though as they do have some custom GQL logic.

Reply:
no breaking change on FE side

Reply:
Thanks for the feedback, I'm removing the "breaking" label then.